vignettes/including-general-data-in-teal.Rmd

```diff
@@ -76,3 +76,5 @@ if (interactive()) {
   shinyApp(app$ui, app$server)
 }
 ```
+
+For more information about preprocessing, reproducibility, relationships between datasets and DDL, please refer to the `teal.data.` [vignette](https://insightsengineering.github.io/teal.data)
```

Comment:
```suggestion
For more information about preprocessing, reproducibility, relationships between datasets and DDL, please refer to the [`teal.data` package](https://insightsengineering.github.io/teal.data)
```

Reply:
Or point explicitly to the vignette...
![image](https://user-images.githubusercontent.com/15201933/181482982-67ba27e3-edc0-4253-b9cb-1569c921dbae.png)

Reply:
Same in other vignettes

Reply:
done.